examples/fishjam-chat/app.json

```diff
@@ -45,7 +47,15 @@
           }
         }
       ],
-      ["./plugin/build/with-local-paths-for-native-packages.js"] // Only relevant for local development of packages/ios-client and packages/android-client
```

Comment:
**issue:** Why remove the comment? `app.json` is [JSON5](https://json5.org/) under the hood, so this is valid syntax.